@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -733,23 +889,39 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
                                 );
                                 line.push(tmpV3.x, tmpV3.y, tmpV3.z);
                             }
+
+                            if (needIndividualLineSegments && line.length > 0 && ringLength > 0) {
+                                // Scale down each individual segment to range [0..1] for the whole
+                                // line.
+                                segmentOffsets!.forEach((offs, index) => {
+                                    segmentOffsets![index] = offs / ringLength;
```

Comment:
Maybe multiply with invRingLenght - this may be pre-computed, in theory should be faster...

Reply:
I never saw a difference in JS/TS with regard to performance of multiplication vs. division. The theoretical difference is not visible anymore with all the overhead and the speed of accessing RAM.